src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -998,6 +1001,12 @@ namespace DataHeatBalance {
 	}
 
 	void
+	ZoneData::SetWindDirAt( Real64 const fac )
```

Comment:
It's used for initializing zone level local var for wind dir.

Initializing in HeatBalanceManager.cc. Same routine as zone level wind speed.

```
    // X Luo Added 07/19/2017
    // set zone level wind dir to global value
    SetZoneWindSpeedAt();
    SetZoneWindDirAt();
```

```
void
SetZoneWindDirAt()
{
	// Using/Aliasing
	Real64 const fac( DataEnvironment::WindDir );
	for ( auto & zone : Zone ) {
		zone.SetWindDirAt( fac );
	}
}

```

And then this value can be overridden with local node value or EMS value if defined. If not, the value would still be set the same with the global input